### DISCUSSION TITLE
如何返回一个 json

### DISCUSSION THREAD

**Discussion by dddrop** [ant-design/ant-design#29074] [Q&A]
在对应iOS的universal links时，需要以json的形式返回 /.well-known/apple-app-site-association文件。

> 例如：twitter.com/.well-known/apple-app-site-association

想请问如何配置。

**Comment by yoyo837** [comment]
不太明白什么诉求以及和antd的关系。

**Reply by dddrop** [reply]
自己理解错了。不好意思。
